ngraph/src/ngraph/pass/visualize_tree.cpp

```diff
@@ -348,6 +349,62 @@ static std::string pretty_partial_shape(const PartialShape& shape)
     return ss.str();
 }
 
+template <typename T>
+static std::string pretty_value(const vector<T>& value)
+{
+    std::stringstream ss;
+    bool first = true;
+    for (const auto& i : value)
+    {
+        if (!first)
+            ss << ", ";
+        ss << i;
+        first = false;
+    }
+    return ss.str();
```

Comment:
How do you think, may be it's better to have limit for vectors with size > 100 and replace middle values with "..." 
Because with this option it will print all values from all constants and for models with huge weights it will be impossible to render this graphs.